test/resolve-config.test.js

```diff
@@ -35,6 +40,7 @@ test('Returns user config via environment variables', t => {
       gitlabApiUrl: urlJoin(gitlabUrl, gitlabApiPathPrefix),
       assets,
       milestones,
+      postComments: false,
```

Comment:
if the default value of this option is `false`, do all of these tests need to set explicitly to `false`? i think it could be good to have at least one test that proves it can handle being set to `false`, but it seems appropriate to leave most of the existing tests with the value unset.

Reply:
See https://github.com/semantic-release/gitlab/pull/332#discussion_r830478105 :)